app/models/requests/mediated_page.rb

```diff
@@ -4,7 +4,7 @@
 class MediatedPage < Request
   validate :mediated_page_validator
   validates :destination, presence: true
-  validates :needed_date, presence: true
+  validate :needed_date_is_required
```

Comment:
Yeah, I had the same panic moment but it looks like we're already handling this in a way that will work: https://github.com/sul-dlss/sul-requests/blob/master/app/views/requests/_form.html.erb#L15